api/src/test/java/org/openmrs/module/fgh/mpi/MpiIntegrationProcessorTest.java

```diff
@@ -371,4 +426,94 @@ public void process_shouldProcessAPersonDeleteEvent() throws Exception {
 	public void process_shouldProcessAPatientDeleteEvent() throws Exception {
 	}
 	
+	@Test
+	public void process_shouldInitTheContextWithoutSSL() throws Exception {
```

Comment:
These tests don't make sense, where are you calling the process?

Reply:
Resolved